src/frontend/src/components/EditUser/index.tsx

```diff
@@ -48,37 +54,131 @@ class EditUser extends React.Component<Props, State> {
+        mismatchPasswords: this.state.password !== this.state.confirmPassword
+      }
+    );
+  }
+
   public render() {
     return (
       <main className="mw6 center ph2">
 
         <section className="cf mt4 mb3 pa3 pa4-l bg-white shadow br3">
-          <h1 className="mb4 f4 fw6">Account: Jaire Alexander</h1>
-          <form>
+          <h1 className="mb4 f4 fw6">Account: </h1>
+          <form onSubmit={this.handleSubmit} noValidate={true}>
             <div className="mb4">
               <label htmlFor="name" className="db mb2 fw6">
                 Name *
               </label>
-              <input id="name" required type="text" value="Jaire Alexander" className="w-100 pa3 br2 b--black-20" />
+              <input
+                id="name"
+                name="name"
+                type="text"
+                className="w-100 pa3 br2 b--black-20"
+                required={true}
+                aria-describedby={
+                  this.state.missingName
+                    ? 'name_input_message'
+                    : undefined
+                }
+                aria-invalid={
+                  this.state.missingName
+                    ? true
+                    : false
+                }
+                onChange={this.handleChange}
+              />
             </div>
-
             <div className="mb4">
               <label htmlFor="email" className="db mb2 fw6">
                 Email *
               </label>
-              <input id="email" required type="email" value="jalexander@example.com" className="w-100 pa3 br2 b--black-20" />
+              <input
+                id="email"
+                name="email"
+                type="email"
+                className="w-100 pa3 br2 b--black-20"
+                required={true}
+                aria-describedby={
+                  this.state.invalidEmail
+                    ? 'email_message'
+                    : undefined
+                }
+                aria-invalid={
+                  this.state.invalidEmail
+                    ? true
+                    : false
+                }
+                onChange={this.handleChange}
+              />
+            </div>
+            <div className="mb4">
+              <label htmlFor="password" className="db mb2 fw6">
+                Password
+              </label>
+              <input
+                id="password"
+                name="password"
+                type="password"
+                className="w-100 pa3 br2 b--black-20"
+                required={true}
+                aria-describedby={
+                  this.state.missingPassword
+                    ? 'password_input_message'
+                    : this.state.invalidPassword
+                    ? 'password_message'
+                    : undefined
+                }
+                aria-invalid={
+                  this.state.missingPassword || this.state.invalidPassword
```

Comment:
I'm pretty sure this ternery operator can be thrown away and you could just use `aria-invalid={ this.state.missingPassword || this.state.invalidPassword}`

Reply:
Thanks for catching that! 
I went through and removed all the unnecessary ternery operators in those attributes.